### PR TITLE
rsstail: add page

### PR DESCRIPTION
**Pull request by melgu**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/rsstail.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# rsstail
+
+> Basically tail for RSS feeds.
```
Comment (agnivade) [inline pages/common/rsstail.md]: We prefer a formal language in our pages. Let us remove "Basically". And put tail in backticks.

File: pages/common/rsstail.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Show feed of given url and wait for new entries:
+
+`rsstail -u {{url}}`
+
+- Show feed with latest item at the bottom:
```
Comment (agnivade) [inline pages/common/rsstail.md]: Let us use the word "reverse" in the description, so that it matches with `-r`

File: pages/common/rsstail.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`rsstail -r -u {{url}}`
+
+- Show feed including publication date and link:
```
Comment (agnivade) [inline pages/common/rsstail.md]: Just mention - "Include publication date and link"

File: pages/common/rsstail.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`rsstail -pl -u {{url}}`
+
+- Show feed and set update interval:
```
Comment (agnivade) [inline pages/common/rsstail.md]: "Set update interval" is fine

**Comment by melgu** [comment]
@agnivade The description of the reverse order is wrong. With '-r' newer items are at the bottom. Normally they would be at the top.

**Comment by agnivade** [comment]
Interesting, so the first command updates the entries at top ? Hmm .. that is not `tail` like behavior.

**Comment by melgu** [comment]
I will check that again. I took the ‚tail‘ part from the official Website. I’ll check the standard behavior again (so if new entries are added at the top or bottom). But for the display of already existing entries, the description is correct (with -r putting the newest one at the bottom).

**Comment by melgu** [comment]
Without `-r`:
First, newest ones at the top. Later ones still appear at the bottom.
```
> rsstail -u http://lorem-rss.herokuapp.com/feed -i 10
Title: Lorem ipsum 2018-08-22T13:56:00+00:00
Title: Lorem ipsum 2018-08-22T13:55:00+00:00
Title: Lorem ipsum 2018-08-22T13:54:00+00:00
Title: Lorem ipsum 2018-08-22T13:53:00+00:00
Title: Lorem ipsum 2018-08-22T13:52:00+00:00
Title: Lorem ipsum 2018-08-22T13:51:00+00:00
Title: Lorem ipsum 2018-08-22T13:50:00+00:00
Title: Lorem ipsum 2018-08-22T13:49:00+00:00
Title: Lorem ipsum 2018-08-22T13:48:00+00:00
Title: Lorem ipsum 2018-08-22T13:47:00+00:00
Title: Lorem ipsum 2018-08-22T13:57:00+00:00
Title: Lorem ipsum 2018-08-22T13:58:00+00:00
Title: Lorem ipsum 2018-08-22T13:59:00+00:00
```
<br>

Now with `-r`:
Latest at the bottom. Later ones also appear at the bottom.
```
> rsstail -ru http://lorem-rss.herokuapp.com/feed -i 10
Title: Lorem ipsum 2018-08-22T13:54:00+00:00
Title: Lorem ipsum 2018-08-22T13:55:00+00:00
Title: Lorem ipsum 2018-08-22T13:56:00+00:00
Title: Lorem ipsum 2018-08-22T13:57:00+00:00
Title: Lorem ipsum 2018-08-22T13:58:00+00:00
Title: Lorem ipsum 2018-08-22T13:59:00+00:00
Title: Lorem ipsum 2018-08-22T14:00:00+00:00
Title: Lorem ipsum 2018-08-22T14:01:00+00:00
Title: Lorem ipsum 2018-08-22T14:02:00+00:00
Title: Lorem ipsum 2018-08-22T14:03:00+00:00
Title: Lorem ipsum 2018-08-22T14:04:00+00:00
Title: Lorem ipsum 2018-08-22T14:05:00+00:00
Title: Lorem ipsum 2018-08-22T14:06:00+00:00
```

The version without `-r` only seems to be reasonable when one is using `-1`.

**Comment by agnivade** [comment]
Yes, let's change it to reflect that.

**Comment by melgu** [comment]
@agnivade Do you have an idea how to best describe the effect?
Isn't this too long while still not being specific enough?
> First, the newest item is at the top. Later ones still appear at the bottom.

**Comment by sbrl** [comment]
@melgu Perhaps it could be described as _chonological order_ and _reverse chronological order_?

**Comment by melgu** [comment]
@sbrl Thanks.

**Pull request merged**